samples/experimental/java/README.md

```diff
@@ -0,0 +1,60 @@
+# Run Playwright Java tests with Microsoft Playwright Testing (Experimental)
+
+This sample demonstrates how to run Playwright Java tests at scale using Microsoft Playwright Testing. It showcases the benefits of accelerating test suite completion by leveraging more parallel cloud browsers.
+
+**Note** - You won't be able to get the support for reporting and authentication to the service from the client running the tests via Entra ID.
+
+If you have not yet created a workspace, please follow the [Get Started guide](../../../README.md#get-started)
+
+## Sample setup
+
+1. Clone this sample:
+
+    ```bash
+    git clone https://github.com/microsoft/playwright-testing-service
+    cd playwright-testing-service/samples/experimental/java
+    ```
+
+1. Install dependencies:
+
+    ```bash
+    mvn install
+    ```
+
+1. Set up Authentication using Access Tokens:
+
+    Currently, only access tokens are supported for Java. See [Set up authentication using access tokens](../../../../README.md#set-up-authentication-using-access-tokens)
+
+1. Set access token generated above as environment variable for your project:
+
+    ```bash
+    PLAYWRIGHT_SERVICE_ACCESS_TOKEN= # Paste Access Token value from previous step
+    ```
+
+1. Set up environment:
+
+    In the [Playwright portal](https://aka.ms/mpt/portal), copy the command under **Add region endpoint in your set up** and set the following environment variable:
+
+    ```bash
+    PLAYWRIGHT_SERVICE_URL= # Paste region endpoint URL
+    ```
+
+## Run tests
+
+Run Playwright tests against browsers managed by the service using the configuration you created above. You can run up to 100 parallel workers with the service
+
+```bash
+mvn compile exec:java -D exec.mainClass="org.example.App"
```

Comment:
can we provide a way to pass on worker count ?

Reply:
or add a link from OSS around parallisation

Reply:
Java has ways on how you use it:

1. Library mode, what this PR is about. There is no parallelism - its just a script you run.
2. Test Runner mode [via junit](https://playwright.dev/java/docs/junit#running-tests-in-parallel). There we provide some custom integrations to run a browser/context/page.

I recommend to also provide a way for junit, probably similar to how we do it in other languages, that the user can specify / override `connectOptions`.